pkg/mtls/sds/subscriber.go

```diff
@@ -126,7 +126,7 @@ func (subscribe *SdsSubscriber) sendRequestLoop(sdsStreamClient *SdsStreamClient
 			for {
 				err := subscribe.sendRequest(discoveryReq)
 				if err != nil {
-					log.DefaultLogger.Alertf("sds.subscribe", "[xds] [sds subscriber] send sds request fail , resource name = %v", name)
+					log.DefaultLogger.Alertf("sds.subscribe.request", "[xds] [sds subscriber] send sds request fail , resource name = %v", name)
```

Comment:
should be used `log.DefaultLogger.Alertf("sds.subscribe.request [xds] [sds subscriber] send sds request fail , resource name = %v", name)`  instead of  `log.DefaultLogger.Alertf("sds.subscribe.request", "[xds] [sds subscriber] send sds request fail , resource name = %v", name)`, otherwise there will be logging "xx !(EXTRA string= xx"

Reply:
no, alertf is different